astropy/tests/helper.py

```diff
@@ -461,6 +460,7 @@ def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=None,
     :func:`numpy.testing.assert_allclose`.
     """
     import numpy as np
+    from ..units.quantity import _unquantify_allclose_arguments
     np.testing.assert_allclose(*_unquantify_allclose_arguments(actual, desired,
```

Comment:
Instead of using a private function from astropy.units, why not just do:

```
assert quantity_allclose(...)
```

?